### DISCUSSION TITLE
Is localhost CORS supported on Parcel 2 ?

### DISCUSSION THREAD

**Discussion by Standaa** [parcel-bundler/parcel#6118] [Q&A]
Hello everybody, it looks like I have a CORS issue when making a localhost json rpc call from the parcel development web server.
Everything works fine on parcel-bundler and I can see an OPTIONS pre-flight being made.
On Parcel 2, however, the request is not even logged in my browser's network calls.

I am therefore wondering whether CORS headers are set on parcel 2 ?

**Comment by Standaa** [comment]
Bumping as i am stuck on Parcel 1 for the time being.

**Comment by mischnic** [comment]
These are the headers that are set on Parcel 2:

https://github.com/parcel-bundler/parcel/blob/031da4927b4e4b3d102be5d59402768243e06693/packages/reporters/dev-server/src/Server.js#L33-L43

Can you give a small example what exactly you're running and the error you get?

**Reply by bminer** [reply]
Cross-origin isolation headers were also added in #6404, and this issue explains how to disable them: https://github.com/parcel-bundler/parcel/issues/6220#issuecomment-885837128
